### PR TITLE
Add VPC related options in NsxConfig

### PR DESCRIPTION
**Pull request by lxiaopei**
Add VPC related options in NsxConfig.

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -201,6 +216,18 @@ func (vcConfig *VCConfig) validate() error {
 	return nil
 }
 
+func (wcpConfig *WCPConfig) validate() error {
+	enableVPC := wcpConfig.EnableWCPVPCNetwork
+	if enableVPC {
```
Comment (seanpang-vmware) [inline pkg/config/config.go:221]: if VPC is not enabled, do we need to pass the validation here?

Comment (lxiaopei) [reply pkg/config/config.go:221]: no, if VPC is not enabled, no validation needed.

**Review by seanpang-vmware — APPROVED** [review]

File: pkg/config/config.go [hunk 2]
```diff
@@ -123,6 +127,10 @@ func NewNSXOperatorConfigFromFile() (*NSXOperatorConfig, error) {
 	if err != nil {
 		return nil, err
 	}
+	err = cfg.Section("wcp").MapTo(nsxOperatorConfig.WCPConfig)
```
Comment (dantingl) [inline pkg/config/config.go:130]: I guess no need "wcp" section?

File: pkg/config/config.go [hunk 3]
```diff
@@ -151,6 +159,9 @@ func (operatorConfig *NSXOperatorConfig) validate() error {
 	if err := operatorConfig.NsxConfig.validate(); err != nil {
 		return err
 	}
+	if err := operatorConfig.WCPConfig.validate(); err != nil {
```
[the remark this answers is not in the document]
Comment (lxiaopei) [reply pkg/config/config.go:162]: removed here.

File: pkg/config/config.go [hunk 4]
```diff
@@ -224,6 +228,14 @@ func (nsxConfig *NsxConfig) validate() error {
 		log.Error(err, "validate NsxConfig failed", "thumbprint count", tpCount, "manager count", mCount)
 		return err
 	}
+	enableVPC := coeConfig.EnableVPCNetwork
+	if enableVPC {
+		if nsxConfig.DefaultProject == "" || len(nsxConfig.ExternalIPv4Blocks) == 0 {
+			err := errors.New("nsxt_project is none or external_ipv4_blocks is empty")
+			log.Error(err, "validate WCPConfig failed")
```
Comment (dantingl) [inline pkg/config/config.go:235]: validate VPC config failed

Comment (lxiaopei) [reply pkg/config/config.go]: changed

**Review by seanpang-vmware — APPROVED** [review]
LGTM

**Pull request merged**